yt_dlp/extractor/common.py

```diff
@@ -1659,11 +1659,9 @@ def _search_nuxt_data(self, webpage, video_id, context_name='__NUXT__', *, fatal
         if js is None:
             return {}
 
-        args = dict(zip(arg_keys.split(','), arg_vals.split(',')))
-
-        for key, val in args.items():
-            if val in ('undefined', 'void 0'):
-                args[key] = 'null'
+        args = dict(zip(arg_keys.split(','),
+                        [json.dumps(v) for v in self._parse_json(f'[{arg_vals}]',
+                         video_id, transform_source=js_to_json, fatal=fatal) or []]))
```

Comment:
```suggestion
        args = dict(zip(arg_keys.split(','), map(json.dumps, self._parse_json(
            f'[{arg_vals}]', video_id, transform_source=js_to_json, fatal=fatal) or ())))
```
I think we don't need to create another intermediate list. Yours works just fine though.